File: backendworker/NHSOnline.Backend.Auth/CitizenId/Models/UserInfo.cs
using System;
using System.IdentityModel.Tokens.Jwt;
using Newtonsoft.Json;

namespace NHSOnline.Backend.Auth.CitizenId.Models
{
    [Serializable]
    public class UserInfo
    {
        [JsonProperty(JwtRegisteredClaimNames.GivenName)]
        public string GivenName { get; set; }

        [JsonProperty(JwtRegisteredClaimNames.FamilyName)]
        public string FamilyName { get; set; }

        [JsonProperty("gp_integration_credentials")]
        public GpIntegrationCredentials GpIntegrationCredentials { get; set; }

        [JsonProperty("nhs_number")]
        public string NhsNumber { get; set; }

        [JsonProperty("im1_token")]
        public string Im1ConnectionToken { get; set; }

        [JsonProperty(JwtRegisteredClaimNames.Sub)]
        public string Subject { get; set; }

        [JsonProperty(JwtRegisteredClaimNames.Birthdate)]
        public string Birthdate { get; set; }

        [JsonProperty("identity_proofing_level")]
        public string IdentityProofingLevel { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/citizenId/models/UserInfo.kt
package mocking.citizenId.models

import com.google.gson.annotations.SerializedName

data class UserInfo (
        @SerializedName("given_name")
        var GivenName: String,

        @SerializedName("family_name")
        var FamilyName: String,

        @SerializedName("gp_integration_credentials")
        var GpIntegrationCredentials: GpIntegrationCredentials,

        @SerializedName("gp_registration_details")
        var GpRegistrationDetails: GpRegistrationDetails,

        @SerializedName("nhs_number")
        var NhsNumber: String,

        @SerializedName("im1_token")
        var Im1ConnectionToken: String?,

        @SerializedName("sub")
        var Subject: String,

        @SerializedName("birthdate")
        var Birthdate: String,

        @SerializedName("identity_proofing_level")
        var IdentityProofingLevel: String?,

        @SerializedName("email")
        val Email: String
)
